packages/components/src/interfaces/complex/KeyStatistics.ts

```diff
@@ -33,6 +33,7 @@ export const KeyStatisticsSchema = Type.Object(
         title: "Link destination",
         description: "When this is clicked, open:",
         format: "link",
+        linkTypes: ["page"],
```

Comment:
wondering if we should make this code a type on its own because `linkTypes` is a custom prop that's unusual of that of String